scenarios/kubernetes_e2e.py

```diff
@@ -60,13 +65,10 @@ def main(args):
     if not os.path.isdir(artifacts):
         os.makedirs(artifacts)
 
-    e2e_image_tag = 'v20170104-9031f1d'
-    e2e_image_tag_override = '%s/hack/jenkins/.kubekins_e2e_image_tag' % workspace
-    if os.path.isfile(e2e_image_tag_override):
-        with open(e2e_image_tag_override) as tag:
-            e2e_image_tag = tag.read()
-
-    # exec
+    try:  # Pull a newer version if one exists
+        check('gcloud', 'docker', '--', 'pull', kubekins(args.tag))
```

Comment:
Does docker pull work? I guess the image is public so we don't need to authenticate?